dataactvalidator/config/sqlrules/sqlRules.csv

```diff
@@ -161,4 +161,6 @@ D42,"PrimaryPlaceOfPerformanceForeignLocationDescription must be blank for domes
 D43,"When PrimaryPlaceOfPerformanceCountryCode is not USA, PrimaryPlaceOfPerformanceCongressionalDistrict must be blank.","When PrimaryPlaceOfPerformanceCountryCode is not USA, PrimaryPlaceOfPerformanceCongressionalDistrict must be blank.",n,detached_award,fatal,,d43_detached_award_financial_assistance_1
 D43,"If no PrimaryPlaceOfPerformanceZIP+4 is provided, a PrimaryPlaceOfPerformanceCongressionalDistrict must be provided.","If no PrimaryPlaceOfPerformanceZIP+4 is provided, a PrimaryPlaceOfPerformanceCongressionalDistrict must be provided.",n,detached_award,fatal,,d43_detached_award_financial_assistance_2
 D43,"If PrimaryPlaceOfPerformanceCongressionalDistrict is provided it must be valid in the state indicated by PrimaryPlaceOfPerformanceCode.","If PrimaryPlaceOfPerformanceCongressionalDistrict is provided it must be valid in the state indicated by PrimaryPlaceOfPerformanceCode.",n,detached_award,fatal,,d43_detached_award_financial_assistance_3
+D44,"Invalid DUNS number","Invalid DUNS number",n,detached_award,warning,,d44_detached_award_financial_assistance

```

Comment:
If you have more than one part, they should just be `_1`, `_2`, etc. You shouldn't have any un-numbered parts. This applies for your file names and the test files as well.